### PR TITLE
Drop old Map Reduce code

### PR DESCRIPTION
**Pull request by milleruntime**

### CODE REVIEW

File: hadoop-mapreduce/pom.xml [hunk 1]
```diff
@@ -94,7 +94,7 @@
             </goals>
             <configuration>
               <includes>
-                <include>org[.]apache[.]accumulo[.]core[.]client[.]mapred(?:uce)?[.].*</include>
+                <include>org[.]apache[.]accumulo[.]hadoop[.]mapred(?:uce)?[.].*</include>
```
Comment (ctubbsii) [inline hadoop-mapreduce/pom.xml:97]: Was this previously not analyzing classes in this module? That's weird.

**Review by ctubbsii — COMMENTED** [review]
Looks good overall. I think you've already moved any old tests to the new M/R framework, but it's possible there could be some that were missed. This would be good to get into 3.0.

**Comment by milleruntime** [comment]
> I think you've already moved any old tests to the new M/R framework, but it's possible there could be some that were missed.

I moved `MapReduceIT` in this PR. There are 2 other ITs that had tests that I dropped: `SslIT` and `SslWithClientAuthIT`. I made an attempt to migrate them to `MapReduceIT` since they were just calling the `runTest` method in that IT. But I couldn't get the SSL to work with MapReduceIT and wasn't sure if it was worth the effort.

**Comment by ctubbsii** [comment]
I merged main into this to resolve conflicts, and am marking it ready for review. I'm not sure what else needs to be done for this, but that can be ironed out in the review.

**Comment by ctubbsii** [comment]
Full ITs passed.

**Pull request merged**